diff --git a/dist/esm/src/compiler.js b/dist/esm/src/compiler.js
index 776de15ff2211f40f56884a1df67899e479c8830..878829a9d7e22908bc649c01d2f993c52c3f8e32 100644
--- a/dist/esm/src/compiler.js
+++ b/dist/esm/src/compiler.js
@@ -15,7 +15,7 @@ import stripJsonComments from 'strip-json-comments';
 import { ReflectionOp, TypeNumberBrand } from '@deepkit/type-spec';
 import { Resolver } from './resolver.js';
 import { knownLibFilesForCompilerOptions } from '@typescript/vfs';
-import * as micromatch from 'micromatch';
+import micromatch from 'micromatch';
 // don't use from @deepkit/core since we don't want to have a dependency to @deepkit/core
 export function isObject(obj) {
     if (obj === null) {
